digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/linearasset/OracleLinearAssetDao.scala

```diff
@@ -63,18 +69,20 @@ class OracleLinearAssetDao(val vvhClient: VVHClient, val roadLinkService: RoadLi
 
   private def addCountsFor(unknownLimitsByMunicipality: Map[String, Map[String, Any]]): Map[String, Map[String, Any]] = {
     val unknownSpeedLimitCounts =  sql"""
-      select name_fi, s.administrative_class, count(*)
+      select name_fi, s.administrative_class, count(*), m.id
       from unknown_speed_limit s
       join municipality m on s.municipality_code = m.id
-      group by name_fi, administrative_class
-    """.as[(String, Int, Int)].list
+      group by name_fi, administrative_class, m.id
+    """.as[(String, Int, Int, Int)].list
 
     unknownLimitsByMunicipality.map { case (municipality, values) =>
       val municipalityCount = unknownSpeedLimitCounts.find(x => x._1 == municipality && x._2 == Municipality.value).map(_._3).getOrElse(0)
       val stateCount = unknownSpeedLimitCounts.find(x => x._1 == municipality && x._2 == State.value).map(_._3).getOrElse(0)
       val privateCount = unknownSpeedLimitCounts.find(x => x._1 == municipality && x._2 == Private.value).map(_._3).getOrElse(0)
+      val municipalityId = unknownSpeedLimitCounts.find(x => x._1 == municipality).map(_._4).getOrElse(0)
 
       val valuesWithCounts = values +
+        ("id" ->  municipalityId) +
```

Comment:
Switch the id key to municipalityId key to avoid confusion with other ids like asset id for example